app/src/main/java/ch/epfl/sdp/peakar/points/LineOfSight.java

```diff
@@ -152,11 +153,13 @@ private boolean isVisible(POIPoint poiPoint, Pair<Integer, Integer> userIndexes,
             // Slope error reached limit, time to
             // increment y and update slope error.
```

Comment:
I think the comment was describing the old algorithm.
```suggestion
            // increment or decrement y and update slope error.
```